go/cloud-query/cmd/args/args.go

```diff
@@ -125,7 +125,7 @@ func DatabaseConnectionTTL() time.Duration {
 		return defaultDatabaseConnectionTTL
 	}
 
-	if *argDatabaseConnectionTTL < 1*time.Minute {
+	if *argDatabaseConnectionTTL < 10*time.Minute {
```

Comment:
this should be way longer, like 2-5h, we'll continuously extract from the connection once an hour, so if its longer than that it likely no longer exists